lib/lotus/action/cookie_jar.rb

```diff
@@ -131,6 +137,7 @@ def extract(env)
         hash   = env[COOKIE_HASH_KEY] ||= {}
         string = env[HTTP_HEADER]
 
+        return if hash.has_key? RACK_SESSION_KEY
```

Comment:
@AlfonsoUceda does it means that if `rack.session` key, the cookie jar will appear as empty? Can we improve the tests and let the actions to set both cookies and sessions values? Thank you :+1: